decidim-assemblies/spec/services/decidim/assemblies/searchable_assembly_resource_spec.rb

```diff
@@ -15,7 +15,8 @@ module Decidim
           subtitle: Decidim::Faker::Localized.name,
           short_description: Decidim::Faker::Localized.sentence,
           description: description1,
-          users: [author]
+          users: [author],
+          is_transparent: false
```

Comment:
As long as the assembly isn't private this shouldn't matter, right?

Reply:
Actually this can be discussed here: #13560

Reply:
It does not matter, but this file uses the spec below. Since we change the visibility rules to include transparency, this spec, starts to fail, as the is_transparent field is set to true in the factory.: 
https://github.com/decidim/decidim/blob/606a187e804e4cfd4a7e0bd9784fae2803eb5ea7/decidim-core/lib/decidim/core/test/shared_examples/searchable_participatory_space_examples.rb#L61-L69